src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeFindingPositiveRiskCellConfigurator.swift

```diff
@@ -81,6 +79,13 @@ final class HomeFindingPositiveRiskCellConfigurator: HomeRiskCellConfigurator {
 		cell.topContainer.accessibilityTraits = [.button, .header]
 	}
 
+	// MARK: Hashable
+
+	func hash(into hasher: inout Swift.Hasher) { }
```

Comment:
This class doesn't have and also doesn't depend on any properties, so can't make difference between 2 instances of this class and it's ok here, because we have only one instance of this class in our logic.

We don't need and cannot call `super`, because this class has no parent.